.env

```diff
@@ -2,3 +2,4 @@ VERSION='1.0-rc'
 APPNAME='Snowball'
 
 NETWORK=mainnet
+PRIVATENODE='https://node.snowapi.net'
```

Comment:
we prob want to not show this in public, can we add it to our vercel env vars instead? or use `.env` file

Reply:
I didnt realize this is the .env file hehe, we should ad this file to .ignore

Reply:
well, it gonna show for the users in the console anyway, it's not a big deal I think